### ISSUE TITLE
Why using namespace jamesiarmes\PEWS

### ISSUE THREAD

**Issue by nvanheuverzwijn** [Garethp/php-ews#24]
This is not really an issue.. But I wonder, why using the namespace `\jamesiarmes\PEWS\` instead of simply `\PEWS\` or even `\ews\` ?

I see in your 0.8 version that you intend to replace `jamesiarmes` part with `garethp`. Any practical reason for doing this ? Usually, developpers don't use multiple library that does the same thing.

I honestly think that using `\ews\` | `\EWS\` would be a better idea. The P in pews feels redundant because this is a php lib so we already know that this is php.

Much regards,

**Comment by Garethp** [comment]
Originally this was a fork of `\jamesiarmes\PEWS\`, and it wasn't a complete overhaul of the project, that happened bit by bit. It started with the `Type` class allowing you to build from an array, so that I could do commands without having to spend a hundred or so lines constructing a request, moved to creating simple use-cases for Calendar, then Mail and so on. Eventually, I ended up generating all of my `Type\`'s from the WSDL file for PSR-2 compatibility and other delicious features such as being able to set a Calanders time from a `DateTime` object (I can't remember if that's still in though...).

Originally I was keeping it in hopes that it would get merged up. Rather than a whole new library, I wanted to improve the original one for a larger audience. But james never contacted me back, and I ended up doing a piece by piece rewrite until it wouldn't be compatible any way...

As for the new namespace, it'll be `garethp/ews`, not `garethp/PEWS`, so I am dropping the `p`. The `p` in `garethp` is from my username. As to why it's `garethp/ews` and not just `ews`, I don't feel right poluting a top level namespace like that. There might be other libraries that do other things with that name, but probably not. I just like namespacing my work under my name I guess. Plus I can then move my packagist package to `garethp/ews` and have parity between the namespace and the composer install command

```
composer install garethp/ews

<?php

use garethp/ews;
```

That's the aim
